ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/ethernet-interfaces.yml

```diff
@@ -1410,3 +1410,147 @@ ethernet_interfaces:
             traffic_allow_vlan: 10
             traffic_allow_access_list: acl1
       mac_based_access_list: true
+
+  - name: Ethernet105
```

Comment:
Please use the interface from 73 instead of 105 and so on.

Reply:
Fixed